app/helpers/products_helper.rb

```diff
@@ -43,7 +43,7 @@ def product_url_hint(product)
   private
 
   def public_calendar_options(product)
-    if current_facility.show_instrument_availability?
+    if current_facility&.show_instrument_availability?
```

Comment:
does `current_facility&.show_instrument_availability? || product.facility.show_instrument_availability?` do the trick?

Reply:
The issue is we only want to call `product.facility` if `current_facility` is not present, rather than any time `current_facility&.show_instrument_availability?` evaluates to false. Does that make sense?

Reply:
Yup, that does make sense, thanks.